### PR TITLE
Random changes

### PR DESCRIPTION
**Pull request by wm4**
The usual minor fixes.

(Will probably push more to this in the coming days.)

### CODE REVIEW

File: audio/out/ao_sdl.c [hunk 1]
```diff
@@ -124,8 +123,10 @@ static int init(struct ao *ao)
     }
     desired.freq = ao->samplerate;
     desired.channels = ao->channels.num;
-    desired.samples = MPMIN(32768, ceil_power_of_two(ao->samplerate *
-                                                     priv->buflen));
+    if (priv->buflen) {
+        desired.samples = MPMIN(32768, ceil_power_of_two(ao->samplerate *
+                                                         priv->buflen));
+    };
```
Comment (sfan5) [inline audio/out/ao_sdl.c]: leftover `;`

**Comment by dnmTX** [comment]
@wm4  i'm actually using --load-scripts=no just to skip some unnecessary checks(i don't use any additional scripts anyway) without worrying about the built in --osc.... script(on which i did some changes) or any others in the @shinchiro git.Now if i continue using it it will be a one big mess i suppose.I jut need to know what prompted you to change it cause until now it's been working just great the way it is.Thanks.

**Comment by Hrxn** [comment]
What do you mean by unnecessary checks?
I don't see the point, care to elaborate?

**Comment by wm4** [comment]
I guess I'll drop the load-scripts commit...

**Pull request merged**